conductr_cli/sandbox_run.py

```diff
@@ -33,171 +33,26 @@ def __init__(self, vm_type):
 def run(args):
     """`sandbox run` command"""
 
-    pull_image(args)
-    features = collect_features(args.features, args.image_version)
-    container_names = scale_cluster(args, features)
+    is_conductr_v1 = major_version(args.image_version) == 1
+    is_conductr_dev = major_version(args.image_version) == 0
```

Comment:
As per my comment, I don't think we really need to worry about dev mode - just follow the README and set things up manually as a ConductR dev.